template/{{cookiecutter.project_name}}/taskcluster/ci/hello/kind.yml

```diff
@@ -0,0 +1,21 @@
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+  - {{cookiecutter.project_slug}}_taskgraph.transforms.hello:transforms
+  - taskgraph.transforms.job:transforms
+  - taskgraph.transforms.task:transforms
+
+task-defaults:
+  worker-type: t-linux-large
```

Comment:
Is this a default worker type that is compatible with both the Firefox CI and Community clusters? (I suppose the same applies to trust domain....and makes me wonder if we need a `cookiecutter.json` file per cluster.

Reply:
Hm, that is a great question and I admit I didn't put much thought into it.

This is definitely specific to `firefox-ci` as it uses the generic `mozilla-t` provisioner. I had considered the need to split this up down the road (once we have `mozilla-taskgraph`). I guess in hindsight, it would have been nice to do that first and *then* implement this properly. But I also wanted to rush it a bit to help out the mobile ci team.

So I think my preference is to land this for now and file an issue to make this more generic (either using more templating now, or blocking on `mozilla-taskgraph` and having that somehow wrap the upstream command). Does that sound ok?